services/checks/funcs.go

```diff
@@ -78,11 +79,24 @@ func formatVersionNum(args ...interface{}) (interface{}, error) {
 		return nil, errors.Errorf("expected int64 argument, got %[1]T (%[1]v)", args[0])
 	}
 
+	if num > 10000000 {
```

Comment:
just add a comment explaining why do we have this condition or for what its used